src/SD_binary/write_example.cpp

```diff
@@ -64,36 +68,59 @@ void setup()
     data_written = 0;
     // Open once, at the beginning 
     myFile = SD.open(filename, FILE_WRITE);
+    // since this is inside the setup() function, I indent it for readability
+    //
+    /*This is where I start the set up I WILL SET THE READINGS VALUES TO ZERO TO BEGIN WITH*/
 
+    VRead = 0; // All variables will start a zero and will change as we continue
+    NumCount = 0;
+    VRec = 0;
+    astore = 0;
 
-/*This is where I start the set up I WILL SET THE READINGS VALUES TO ZERO TO BEGIN WITH*/
-
-VRead = 0; // All variables will start a zero and will change as we continue
-NumCount = 0;
-VRec = 0;
-astore = 0;
-
-
-//Setting the pin modes
-
-pinMode(A0, INPUT);
-
-//The serial begin has already been set so I will no do that
-
-
+    //Setting the pin modes
 
+    pinMode(A0, INPUT);
 
+    //The serial begin has already been set so I will no do that
 }
 
-
 void loop()
 {
-    while( stop == false){ /* Allow for the two serial outputs to not be juggled */
```

Comment:
this didn't make sense -- I already have a condition for data acquisition below.  I was a little concerned the structure/purpose of the ``if(millis() - star...`` wasn't clear, so let me know if there are questions there.

Reply:
remember that the ``loop(`` function is looped continuously after the setup is complete.